src/services/webapi/webapi.service.ts

```diff
@@ -18,17 +18,20 @@ export class WizdomWebApiService implements IWizdomWebApiService {
         this.state.corsProxyFailed = false;      
         this.corsProxy.AddHandler("WizdomCorsProxySuccess", (message) => {
             this.state.corsProxyReady = true;
+            this.state.corsProxyFailed = false;
+            this.corsProxy.HandleMessage("WizdomCorsProxyFailed");
```

Comment:
What does this call to corsproxy failed serve? That will just set the state to failed on every success as far as I can tell?

Except it doesn't because the correct format is `HandleMessage({command: "WizdomCorsProxyFailed"});`